test/src/e2e_vm_tests/test_programs/should_pass/stdlib/u128_pow_test/src/main.sw

```diff
@@ -0,0 +1,47 @@
+script;
+
+use std::u128::*;
+
+fn main() -> bool {
+
+    let mut u_128 = U128::from((0, 7));
+    let mut pow_of_u_128 = u_128.pow(U128::from((0, 2)));
+    assert(pow_of_u_128 == U128::from((0, 49)));
+
+    pow_of_u_128 = u_128.pow(U128::from((0, 3)));
+    assert(pow_of_u_128 == U128::from((0, 343)));
+
+    u_128 = U128::from((0, 3));
+    pow_of_u_128 = u_128.pow(U128::from((0, 2)));
+    assert(pow_of_u_128 == U128::from((0, 9)));
+
+    u_128 = U128::from((0, 5));
+    pow_of_u_128 = u_128.pow(U128::from((0, 2)));
+    assert(pow_of_u_128 == U128::from((0, 25)));
+
+    pow_of_u_128 = u_128.pow(U128::from((0, 7)));
+    assert(pow_of_u_128 == U128::from((0, 78125)));
+
+    u_128 = U128::from((0, 8));
+    pow_of_u_128 = u_128.pow(U128::from((0, 2)));
+    assert(pow_of_u_128 == U128::from((0, 64)));
+
+    pow_of_u_128 = u_128.pow(U128::from((0, 9)));
+    assert(pow_of_u_128 == U128::from((0, 134217728)));
+
+    u_128 = U128::from((0, 10));
+    pow_of_u_128 = u_128.pow(U128::from((0, 2)));
+    assert(pow_of_u_128 == U128::from((0, 100)));
+
+    pow_of_u_128 = u_128.pow(U128::from((0, 5)));
+    assert(pow_of_u_128 == U128::from((0, 100000)));
+
+    u_128 = U128::from((0, 12));
+    pow_of_u_128 = u_128.pow(U128::from((0, 2)));
+    assert(pow_of_u_128 == U128::from((0, 144)));
+
+    pow_of_u_128 = u_128.pow(U128::from((0, 3)));
+    assert(pow_of_u_128 == U128::from((0, 1728)));
+
+    true
+}
```

Comment:
```suggestion
}

```

Minor nit, now that `assert_eq` is in the `std` should we use it here instead? Moreover, instead of using a script here should we use in language tests too?

Reply:
I agree that a unit test is better fitting here. We'll still merge this urgent fix but would certainly be nice to port all the math libraries tests into unit tests.